modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/select_account_entry.jsp

```diff
@@ -54,6 +54,12 @@ SelectAccountEntryManagementToolbarDisplayContext selectAccountEntryManagementTo
 				<%
 				Map<String, Object> data = HashMapBuilder.<String, Object>put(
 					"accountentryid", accountEntryDisplay.getAccountEntryId()
+				).put(
+					"entityid", accountEntryDisplay.getAccountEntryId()
+				).put(
+					"entityname", accountEntryDisplay.getName()
+				).put(
+					"entityroles", StringPool.BLANK
```

Comment:
Roles won't be available here. We need the userId in order to get the roles. This should be set in `account_user/accounts.jsp`.